TransformationSystem/scripts/dirac-production-runjoblocal.py

```diff
@@ -0,0 +1,189 @@
+      if element.text == "DIRAC.WorkloadManagementSystem.Client.DownloadInputData":
+        element.text = "DIRAC.WorkloadManagementSystem.Client.InputDataByProtocol"
+        archive.write(basepath + "/InputSandbox" + str(jobID) + "/jobDescription.xml")
+        S_OK("Job parameter changed from DownloadInputData to InputDataByProtocol.")
+
+  
+def __downloadPilotScripts(basepath, diracpath):
+  """
+  Downloads the scripts necessary to configure the pilot
+  
+  """
+  ### dirac path = /afs/cern.ch/lhcb/software/releases/DIRAC/DIRAC_v6r14p5/DIRAC/WorkloadManagementSystem/PilotAgent
+  shutil.copyfile(str(diracpath) + "/WorkloadManagementSystem/PilotAgent/dirac-pilot.py"   , basepath + "/dirac-pilot.py")
+  shutil.copyfile(str(diracpath) + "/WorkloadManagementSystem/PilotAgent/pilotCommands.py" , basepath + "/pilotCommands.py")
+  shutil.copyfile(str(diracpath) + "/WorkloadManagementSystem/PilotAgent/pilotTools.py"    , basepath + "/pilotTools.py")
+  
+  
+#   #include retry function
+#   out = os.system("wget -P " + basepath + "/ http://lhcbproject.web.cern.ch/lhcbproject/Operations/VM/pilotscripts/LHCbPilotCommands.py")
+#   if not out:
+#     S_OK("LHCbPilotCommands.py script successfully download.\n")
+#   else:
+#     print "LHCbPilotCommands.py script download error.\n"
+#     #DError(errno.ENETUNREACH, "LHCbPilotCommands.py script download error.\n" )
+#   out = os.system("wget -P " + basepath +  "/ http://lhcbproject.web.cern.ch/lhcbproject/Operations/VM/pilotscripts/dirac-pilot.py")
+#   if not out:
+#     S_OK("dirac-pilot.py script successfully download.\n")
+#   else:
+#     print "dirac-pilot.py script download error.\n"
+#     #DError(errno.ENETUNREACH, "dirac-pilot.py script download error.\n" )
+#   out = os.system("wget -P " + basepath +  "/ http://lhcbproject.web.cern.ch/lhcbproject/Operations/VM/pilotscripts/pilotCommands.py")
+#   if not out:
+#     S_OK("pilotCommands.py script successfully download.\n")
+#   else:
+#     print "pilotCommands.py script download error.\n"
+#     #DError(errno.ENETUNREACH, "pilotCommands.py script download error.\n" )
+#   out = os.system("wget -P " + basepath +  "/ http://lhcbproject.web.cern.ch/lhcbproject/Operations/VM/pilotscripts/pilotTools.py")
+#   if not out:
+#     S_OK("pilotTools.py script successfully download.\n")
+#   else:
+#     print "pilotTools.py script download error.\n"
+#     #DError(errno.ENETUNREACH, "pilotTools.py script download error.\n" )
+    
+    
+def __configurePilot(basepath):
+  """
+  Configures the pilot.
+  This method was created specifically for LHCb pilots, more info
+  about othe VOs is needed to make it more general.
+  """
+#   out = os.system("python " + basepath + "/dirac-pilot.py -S LHCb-Production -l LHCb -C dips://lbvobox18.cern.ch:9135/Configuration/Server -N ce.debug.ch -Q default -n DIRAC.JobDebugger.ch -M 1 -E LHCbPilot -X LHCbConfigureBasics,LHCbConfigureSite,LHCbConfigureArchitecture,LHCbConfigureCPURequirements -dd")
```

Comment:
vo = getVO() 
would it be better if I use the:
Extensions().getCSExtensions()[0]
?